site/content/en/docs/getting_started/overview.md

```diff
@@ -0,0 +1,202 @@
+
+- **Self-hosted CVAT** (forever Free of charge): Follow the [**Self-hosted Installation Guide**](/docs/administration/basics/installation/)
+  for setup. We provide [**Enterprise-level support**](https://www.cvat.ai/pricing/on-prem)
+  for this version, **including premium features** like **SSO**, **LDAP**, advanced integrations with
+  [**Roboflow and HuggingFace**](https://www.cvat.ai/post/integrating-hugging-face-and-roboflow-models),
+  and **advanced analytics**. We also offer **professional training** and **24-hour SLA support**.
+
+
+See:
+
+- [Tools and formats](#tools-and-formats)
+  - [Supported formats](#supported-formats)
+  - [Annotation tools](#annotation-tools)
+- [Automated labeling](#automated-labeling)
+- [Useful links](#useful-links)
+- [Integrations](#integrations)
+- [License Information](#license-information)
+- [Get in touch](#get-in-touch)
+
+## Tools and formats
+
+CVAT stands as a comprehensive
+tool for image and video annotation,
+essential for various computer vision tasks.
+It emphasizes user-friendliness,
+adaptability, and compatibility with a range of formats and tools.
+
+### Supported formats
+
+CVAT's supports the following formats:
+
+- **For 3D**: `.pcd`, `.bin`
+- **For image**: everything supported by the Python Pillow library,
+  including formats like `JPEG`, `PNG`, `BMP`, `GIG`, `PPM`and `TIFF`.
+- **For video**: all formats, supported by ffmpeg, including `MP4`, `AVI`, and `MOV`.
+
+For annotation export and import formats, see
+[**Export annotations and data from CVAT**](/docs/manual/advanced/formats/)
+
+### Annotation tools
+
+CVAT offers a wide array of annotation tools,
+each catering to different aspects of image and video labeling:
+
+- **3D Object Annotation**: For annotating objects in three dimensions,
+  providing a comprehensive perspective.
+- **Attribute Annotation Mode**: Allows adding specific
+  attributes to annotations.
+- **Annotation with Rectangles**: Ideal for straightforward
+  object detection tasks.
+- **Annotation with Polygons**: Perfect for detailed
+  segmentation of irregular shapes.
+- **Annotation with Polylines**: Useful for tracing
+  elongated shapes or paths.
+- **Annotation with Ellipses**: Suitable for objects
+  with elliptical shapes.
+- **Annotation with Cuboids**: For 3D representation
+  of objects in two dimensions.
+- **Annotation with Skeletons**: Useful in pose estimation and
+  tracking of articulated objects.
+- **Annotation with Brush Tool**: Allows freehand drawing for
+  detailed and complex annotations.
+- **Annotation with Tags**: Tagging images or video frames for
+  classification or categorization.
+
+These tools make CVAT a versatile platform for a range of annotation
+needs, from basic labeling to complex, multidimensional
+tasks in advanced computer vision projects.
+
+## Automated labeling
+
+CVAT has an automated labeling features,
```

Comment:
Again, strage formating. 80 chars in a line should work.

Reply:
@nmanovic same comment as above... if you mean just formatting here. Or it look different for you when you check it on localhost?

Reply:
Please ask if you cannot interpret the guidelines. It is markdown. It should be readable. When you less than half of a page, it is not BKMs for sure.

len("CVAT has an automated labeling features,") is 40 characters!